snapcraft/plugins/catkin.py

```diff
@@ -140,6 +134,16 @@ def env(self, root):
                     root, 'opt', 'ros', self.options.rosdistro, 'setup.sh'))
         ]
 
+        # The ROS packaging system tools (e.g. rospkg, etc.) don't go
+        # into the ROS install path (/opt/ros/$distro), so we need the
+        # PYTHONPATH to include the dist-packages in /usr/lib as well.
+        try:
+            env.append('PYTHONPATH={0}'.format(common.get_python2_path(root)))
+        except EnvironmentError as e:
+            logger.debug(e)
```

Comment:
It's because this WILL happen when run is called, until the stage-packages have been installed.